examples/avro-cli.py

```diff
@@ -53,17 +56,29 @@ def __init__(self, name=None, favorite_number=None, favorite_color=None):
         # Do *not* include in the serialized object.
         self.id = uuid4()
 
-    def to_dict(self):
+
+class UserConverter(Converter):
+
+    @staticmethod
+    def to_dict(user):
```

Comment:
you're not implementing the methods in the base class here.

Reply:
Same as above took advantage of Python method resolution semantics. Will fix , same for subsequent comments.